src/styles/blocks/antisnake.scss

```diff
@@ -0,0 +1,43 @@
+.snake {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  @media(max-width: 899px) {
+    grid-template-columns: 1fr 1fr;
+  }
+  @media(max-width: 599px) {
+    grid-template-columns: 1fr;
+  }
+
+  &__cell {
+    min-width: $cell-heiht;
```

Comment:
When specifying grid-columns, you can omit width for the grid items. Gird will dictate theirs width

```suggestion
 
```